homeassistant/components/remote/itach.py

```diff
@@ -0,0 +1,118 @@
+"""
+Support for iTach IR Devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/remote.itach/
+"""
+
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+import homeassistant.components.remote as remote
+from homeassistant.const import (
+    DEVICE_DEFAULT_NAME, CONF_NAME, CONF_MAC, CONF_HOST, CONF_PORT,
+    CONF_DEVICES, CONF_FILENAME)
+from homeassistant.components.remote import (
+    PLATFORM_SCHEMA, ATTR_COMMAND)
+
+REQUIREMENTS = ['pyitachip2ir==0.0.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_PORT = 4998
+CONNECT_TIMEOUT = 5000
+
+CONF_MODADDR = 'modaddr'
+CONF_CONNADDR = 'connaddr'
+CONF_COMMANDS = 'commands'
+CONF_DATA = 'data'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_MAC, default=None): cv.string,
```

Comment:
The default value if unset is already None I believe so this default line is unnecessary.